src/live_restore/live_restore_fs.c

```diff
@@ -1215,7 +1216,7 @@ __wt_live_restore_fh_to_metadata(WT_SESSION_IMPL *session, WT_FILE_HANDLE *fh, W
           "%s: Appending live restore bitmap (%s, %" PRIu64 ") to metadata", fh->name,
           (char *)buf.data, lr_fh->destination.nbits);
     } else {
-        WT_ERR(__wt_buf_catfmt(session, meta_string, ",live_restore="));
+        WT_ERR(__wt_buf_catfmt(session, meta_string, ",live_restore=(bitmap=,nbits=-1)"));
```

Comment:
Please put a comment here explaining the -1, it's peculiar to see with no context.

Reply:
[ef81a2a](https://github.com/wiredtiger/wiredtiger/pull/11613/commits/ef81a2ab16ac5e4c2755f08a44ecefa57042ba75)